app/graphql/types/movie_type.rb

```diff
@@ -26,6 +26,16 @@ class MovieType < Base::Object
           null: true,
           description: I18n.t("#{I18N_PATH}.fields.poster")
 
+    field :is_favorite,
```

Comment:
Nit: you can pass `resolver_method: :favorite?`, rename resolver method and remove Naming/PredicateName cop disabling